adhocracy-plus/assets/scss/components/_editpoll.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .editpoll__btn--dark {
-    @extend .btn--bg-tertiary;
+    @extend .btn--light;
```

Comment:
why not using `.btn--light` directly?

Reply:
it comes from a4 and we don't use btn--light in the other apps